index.html

```diff
@@ -2092,319 +1905,526 @@ <h1>Core Representations</h1>
+           6375727665                               #         "curve"
+        67                                          #       text(7)
+           65643235353139                           #         "ed25519"
+        67                                          #       text(7)
+           65787069726573                           #         "expires"
+        74                                          #       text(20)
+           323031392d31322d30315430333a30303a30305a #         "2019-12-01T03:00:00Z"
+        6f                                          #       text(15)
+           7075626c69634b6579426173653634           #         "publicKeyBase64"
+        78 2c                                       #       text(44)
+           716d7a3774704c4e4b4b4b646c376344         #         "qmz7tpLNKKKdl7cD"
+           375062656a4469425670374f4e706d5a         #         "7PbejDiBVp7ONpmZ"
+           62666d633763454b396d673d                 #         "bfmc7cEK9mg="
+  6e                                                #   text(14)
+     61757468656e7469636174696f6e                   #     "authentication"
+  81                                                #   array(1)
+     78 83                                          #     text(131)
+        6469643a6578616d706c653a31324433            #       "did:example:12D3"
+        4b6f6f574d4864727a6377706a626472            #       "KooWMHdrzcwpjbdr"
+        5a733547477145524176636771583362            #       "Zs5GGqERAvcgqX3b"
+        3564707550745061396f743639796577            #       "5dpuPtPa9ot69yew"
+        3b6b65792d69643d6261667972656963            #       ";key-id=bafyreic"
+        756274783577716f336e6f7363346361            #       "ubtx5wqo3nosc4ca"
+        7a726b63746668776436726577657a67            #       "zrkctfhwd6rewezg"
+        70776f6534737769726c733465626468            #       "pwoe4swirls4ebdh"
+        733269                                      #       "s2i"
+</pre>
+
+<section>
+  <h4>Consumption</h4>
+<p>  When consuming DID Documents that are represented as CBOR, in addition to the suggestions in section 3.9 of the CBOR [[RFC7049]] specification for deterministic mappings the following constraints of the DID Document model MUST be followed:  </p> 
+
+  <ul>
+  <li>The keys in every map must be sorted lowest value to highest. Sorting is performed on the bytes of the representation of the keys.</li>
+  <li>Indefinite-length items must be made into definite-length items.</li>
+</ul>
+
+</section>
+
+<section>
+  <h4>CBOR Extensibility</h4>
+
+  <p>In CBOR, one point of extensibility is with the use of CBOR tags. [[RFC7049]]  defines a basic set of data types, as
+    well as a tagging mechanism that enables extending the set of data types supported via an IANA registry [[CBOR-TAG-Registry]]. This allows for tags to enhance the semantic description of the data that follows. 
+  </p>
 
     <section>
-        <h2>
-CBOR
-        </h2>
+      <h2>DagCBOR</h2>
```

Comment:
Are there different production/consumption methods for this, like with JSON vs JSON-LD? If so this should be its own representation section.

Reply:
is this cleaner.  Production with constraints in CDDL and consumption with deterministic ordering.    Should I make additional production and consumption for DagCBOR?